examples/cesium/3d-tiles/app.js

```diff
@@ -2,11 +2,11 @@
 
 import {Vector3} from 'math.gl';
 import {registerLoaders} from '@loaders.gl/core';
-import {DracoWorkerLoader} from '@loaders.gl/draco';
+import {DracoLoader} from '@loaders.gl/draco';
 import {Tileset3D, _getIonTilesetMetadata} from '@loaders.gl/3d-tiles';
 import {Plane} from '@math.gl/culling';
 
-registerLoaders([DracoWorkerLoader]);
```

Comment:
Do we need to fix something with DracoWorkerLoader?

Reply:
yes. we need fix. `DracoWorkerLoader` is not working.